packages/bus/src/RetryStrategies/ExponentialBackoffRetryStrategy.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { ExponentialBackoffRetryStrategy } from './ExponentialBackoffRetryStrategy';
+
+describe('ExponentialBackoffRetryStrategy', () => {
```

Comment:
First of all, you should use fake timers, Secondly, you should reset all mocks after each test. Thirdly, you should be consistent either putting `AAA` comments everywhere or removing them entirely.